savant/config/schema.py

```diff
@@ -232,8 +243,42 @@ def get_element_name(element: Union[DictConfig, PipelineElement]) -> str:
 
 
 @dataclass
-class DrawFunc(PyFunc):
-    """A callable PyFunc that will use an object implementing
+class PyFuncElement(PipelineElement, PyFunc):
+    """A pipeline element that will use an object implementing
+    :py:class:`~savant.base.pyfunc.BasePyFuncPlugin` to apply custom processing to
+    gstreamer buffers.
+
+    For example,
+
+    .. code-block:: yaml
+
+        - element: pyfunc
+          module: module.pyfunc_implementation_module
+          class_name: PyFuncImplementationClass
+    """
+
+    element: str = 'pyfunc'
+    """``"pyfunc"`` is the fixed gstreamer element class for PyFuncElement."""
+
+    def __post_init__(self):
+        kwargs = {}
+        if 'kwargs' in self.properties and self.properties['kwargs']:
+            kwargs = json.loads(self.properties['kwargs'])
+        if self.kwargs:
+            kwargs.update(self.kwargs)
+
+        self.properties.update(
+            {
+                'module': self.module,
+                'class': self.class_name,
+                'kwargs': json.dumps(kwargs),
+            }
+        )
+
+
+@dataclass
+class DrawFunc(PyFuncElement):
```

Comment:
@denisvmedyantsev 
I don't see the point as relevant. It can be an element if there are no certain problems with that. Because it is an element but always placed last. The user doesn't care, but from our perspective, we don't need various blocks to be represented by various concepts without the need. I don't see a need to make DrawFunc differ from PyFunc.